packages/shopify-cart/package.json

```
@@ -18,7 +18,8 @@
     "tsc": "tsc",
     "tsc:production": "tsc --project tsconfig.production.json",
     "build": "vite build && tsc --project tsconfig.production.json && mv dist/src dist/types",
-    "release": "npm run build && changeset publish"
+    "release": "npm run build && changeset publish",
+    "lockfile:update": "npm i --package-lock-only"
```

While we're at it, we can remove the `release` script - this looks like a holdover from when the package lived in a separate repo.

```suggestion
    "lockfile:update": "npm i --package-lock-only"
```